linea-besu-package/versions.env

```diff
@@ -1,7 +1,6 @@
-LINEA_BESU_TAR_GZ=25.11.0-RC1-linea2
-LINEA_SEQUENCER_PLUGIN_VERSION=4.0-rc19
-LINEA_TRACER_PLUGIN_VERSION=beta-v4.0-rc19
-LINEA_FINALIZED_TAG_UPDATER_PLUGIN_VERSION=0.3.0
+LINEA_BESU_TAR_GZ=25.10.0-linea3
+LINEA_SEQUENCER_PLUGIN_VERSION=4.0-rc20
+LINEA_TRACER_PLUGIN_VERSION=beta-v4.0-rc20
```

Comment:
why these changes?

Reply:
Just to align with the version we currently used in prod, another reason is that linea-tracer beta-v4.0-rc19 was somehow deleted, so if not updated this version file would be broken